src/misc/KeyManager.js

```diff
@@ -85,6 +85,8 @@ export const Keys = {
 	C: {code: 67, name: "C"},
 	E: {code: 69, name: "E"},
 	F: {code: 70, name: "F"},
+  	J: {code: 74, name: "J"},
```

Comment:
`J` and `K` should follow after `H` and `I`, shouldn't it? :smile:

Reply:
Yeah, I have no idea what went wrong here :sweat_smile: